apps/code-examples/src/app/code-examples/popovers/dropdown/basic/demo.component.html

```diff
@@ -12,3 +12,17 @@
     </sky-dropdown-item>
   </sky-dropdown-menu>
 </sky-dropdown>
+<br />
+<sky-dropdown buttonType="context-menu" label="Code example context menu">
```

Comment:
A context menu doesn't need a sky-dropdown-button--we render it for them. If they put it there it is just ignored. The first one does need it, since they are customizing the button that opens the dropdown.

Reply:
OK. Thanks for the explanation. I didn't realize that the reason for the difference was customizing the button that opens.